circle.yml

```diff
@@ -595,10 +591,10 @@ jobs:
             rustup toolchain install nightly-x86_64-unknown-linux-gnu
       - run:
           name: Build (debug mode)
-          command: RUSTFLAGS="-Z sanitizer=address" ASAN_OPTIONS=detect_leaks=1 cargo +nightly build --target x86_64-unknown-linux-gnu
+          command: RUSTFLAGS="-Z sanitizer=address" cargo +nightly build --target x86_64-unknown-linux-gnu
```

Comment:
Does asan does anything at compile time? If not we can remove this step.

Reply:
Apparently it does some compile-thing things too: https://github.com/rust-lang/rust/issues/45220

Any we should also pass on sanitize mode to cmake: https://users.rust-lang.org/t/sanitizers-in-mixed-rust-and-c-code/48863/4

Reply:
You should be building fizzy with `-DSANITIZE=address`.